client/sysinfo/src/sysinfo.rs

```diff
@@ -58,9 +162,9 @@ pub(crate) fn benchmark<E>(
 		}
 	}
 
-	let score = ((size * count) as f64 / elapsed.as_secs_f64()) / (1024.0 * 1024.0);
+	let score = Throughput((size * count) as f64 / elapsed.as_secs_f64());
```

Comment:
Why are you just constructing it when we have these constructors: `from_mebibyte` etc.